packages/libp2p/src/connection-manager/constants.ts

```diff
@@ -62,3 +67,13 @@ export const INBOUND_CONNECTION_THRESHOLD = 5
  * @see https://libp2p.github.io/js-libp2p/interfaces/index._internal_.ConnectionManagerConfig.html#maxIncomingPendingConnections
  */
 export const MAX_INCOMING_PENDING_CONNECTIONS = 10
+
+/**
+ * Store as part of the peer store metadata for a given peer, the value for this
+ * key is a stringified number representing the timestamp of the last time a
+ * dial attempted failed with the relevant peer.
+ *
+ * Used to insure we do not endlessly try to auto dial peers we have recently
+ * failed to dial.
```

Comment:
nit, but also a learning op on semantics for me: are the dial attempts failures, or are the connections failures? 

```suggestion
 * Used to insure we do not endlessly try to auto dial peers we have recently
 * failed to connect to.
```

Reply:
An attempt to open a connection is a "dial"?